src-electron/db/query-config.js

```diff
@@ -222,14 +223,16 @@ async function insertOrUpdateAttributeState(
     staticAttribute.defaultValue == 0
   ) {
     let featureMapDefaultValue = staticAttribute.defaultValue
-    let mandatoryFeaturesOnEndpointTypeAndCluster =
+    let featuresOnEndpointTypeAndCluster =
       await queryDeviceType.selectDeviceTypeFeaturesByEndpointTypeIdAndClusterId(
         db,
         endpointTypeId,
         clusterRef
       )
-    let featureMapBitsToBeEnabled =
-      mandatoryFeaturesOnEndpointTypeAndCluster.map((f) => f.featureBit)
+    // only set featureMap bit to 1 for mandatory features
+    let featureMapBitsToBeEnabled = featuresOnEndpointTypeAndCluster
+      .filter((f) => f.conformance == 'M')
```

Comment:
Why do we filter for just Mandatory? Were we doing something wrong before?

Reply:
Before we only loaded mandatory features to the database. Now features with all types of conformance exist, so we restrict to only enable bits for mandatory features by default